Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -266,6 +266,7 @@ extension XCTestCase {
 }
 
 extension ConfigurationTests {
+    // swiftlint:disable:next void_return
     static var allTests: [(String, (ConfigurationTests) -> () throws -> Void)] {
```

Comment:
I'm not sure we can handle this case, so I've disabled the rule. I could change to `(ConfigurationTests) -> Void throws -> Void`, but then https://github.com/realm/SwiftLint/pull/965 would trigger